<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\User\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\bin\Debug\PersonalBudgeting.Tests.dll" total="1" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2014-06-25" time="12:09:11">
  <environment nunit-version="2.6.3.13283" clr-version="2.0.50727.5477" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="C:\Program Files (x86)\NUnit 2.6.3\bin" machine-name="PBIO-2355" user="User" user-domain="PBIO-2355" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Users\User\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\bin\Debug\PersonalBudgeting.Tests.dll" executed="True" result="Failure" success="False" time="0.899" asserts="0">
    <results>
      <test-suite type="Namespace" name="PersonalBudgeting" executed="True" result="Failure" success="False" time="0.880" asserts="0">
        <results>
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="0.880" asserts="0">
            <results>
              <test-suite type="TestFixture" name="RealLifeScenariosTests" executed="True" result="Failure" success="False" time="0.878" asserts="0">
                <results>
                  <test-case name="PersonalBudgeting.Tests.RealLifeScenariosTests.TestCalculateBudget_NormalScenario_Successful" executed="True" result="Failure" success="False" time="0.058" asserts="5">
                    <failure>
                      <message><![CDATA[  Expected: 100360
  But was:  100360.00158071518d
]]></message>
                      <stack-trace><![CDATA[at PersonalBudgeting.Tests.RealLifeScenariosTests.TestCalculateBudget_NormalScenario_Successful() in c:\Users\User\Documents\GitHub\PersonalBudgeting\PersonalBudgeting.Tests\RealLifeScenariosTests.cs:line 42
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>